extensions/wikia/UserLogin/UserLoginSpecialController.class.php

```diff
@@ -114,7 +114,7 @@ private function getAuthenticationResource() {
 	 * Shown for both Special:UserLogin and Special:UserSignup when visited logged in.
 	 */
 	public function loggedIn() {
-		$this->redirectToMercury();
+		$this->redirectToAuthPages();
 	}
 
 	public function loginForm() {
```

Comment:
This method still contains code that handles `POST` requests. Should it be removed?